docs/source/torch.rst

```diff
@@ -270,6 +270,8 @@ Other Operations
 .. autofunction:: trace
 .. autofunction:: tril
 .. autofunction:: triu
+.. autofunction:: argsort
```

Comment:
Shouldn’t `argsort` be in the same category as `sort`? And could you please add this in alphabetical order?